### PR TITLE
Helm release job

### PR DESCRIPTION
**Pull request by dwertent**
Helm-chart for paladin Operator

### CODE REVIEW

File: operator/contractpkg/main.go [hunk 1]
```diff
@@ -129,7 +135,11 @@ func (m *ContractMap) process(name string, b *ContractMapBuild) error {
 		for libName, link := range b.LinkedLibs {
 			link = strings.ReplaceAll(link, "_", "-")
 			requiredBuilds = append(requiredBuilds, link)
-			linkedContracts[libName] = fmt.Sprintf(`{{index .status.resolvedContractAddresses "%s"}}`, link)
+			l := fmt.Sprintf(`{{index .status.resolvedContractAddresses "%s"}}`, link)
+			if helmCompatible {
+				l = fmt.Sprintf("{{`%s`}}", l)
```
Comment (dwertent) [inline operator/contractpkg/main.go:140]: TODO: This should take place in the Makefile (when copying the contracts)

File: operator/charts/paladin-operator/values.yaml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 operator:
   name: paladin-operator
   namespace: paladin
   image:
-    repository: paladin-operator
-    tag: latest
-    pullPolicy: IfNotPresent
+    repository: kaleidoinc/paladin-operator
+    tag: release
+    pullPolicy: Always
```
Comment (peterbroadhurst) [inline operator/charts/paladin-operator/values.yaml:14]: Think this is a nice add for now, but we'll likely need to come back to this as part of wider release management strategy as we go forwards.

File: operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon.yaml [hunk 3]
```diff
@@ -23,12 +23,12 @@ spec:
     {
       "name": "Zeto_Anon",
       "implementation": {
-        "implementation":        "{{ index .status.resolvedContractAddresses "zeto-impl-anon" }}",
-        "depositVerifier":       "{{ index .status.resolvedContractAddresses "zeto-g16-check-hashes-value" }}",
-        "withdrawVerifier":      "{{ index .status.resolvedContractAddresses "zeto-g16-check-inputs-outputs" }}",
-        "batchWithdrawVerifier": "{{ index .status.resolvedContractAddresses "zeto-g16-check-inputs-outputs-batch" }}",
-        "verifier":              "{{ index .status.resolvedContractAddresses "zeto-g16-verifier-anon" }}",
-        "batchVerifier":         "{{ index .status.resolvedContractAddresses "zeto-g16-verifier-anon-batch" }}"
+        "implementation":        '{{`{{ index .status.resolvedContractAddresses "zeto-impl-anon" }}`}}',
```
Comment (peterbroadhurst) [inline operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon.yaml:26]: 🥸 🥸

File: operator/contractpkg/main.go (hunk 1 above)
Comment (peterbroadhurst) [reply operator/contractpkg/main.go:140]: Why do we need the non-helm compatible version here? Surely if this is valid YAML, then let's just do this version

**Review by peterbroadhurst — APPROVED** [review]
Looks great. One comment/question before merge @dwertent

File: operator/charts/paladin-operator/values.yaml (hunk 2 above)
Comment (dwertent) [reply operator/charts/paladin-operator/values.yaml:14]: Definitely!! (same with the paladin image, that will be a nightmare)

File: operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon.yaml (hunk 3 above)
Comment (dwertent) [reply operator/config/samples/core_v1alpha1_transactioninvoke_zeto_register_anon.yaml:26]: Yes, the helm templating failed otherwise!

**Pull request merged**